packages/elements/src/pagination/index.ts

```diff
@@ -637,27 +638,23 @@ export class Pagination extends BasicElement {
    * @returns {void}
    */
   private onKeyDown (event: KeyboardEvent): void {
+
     if (event.defaultPrevented) {
       return;
     }
 
-    const newInputValue = Number(this.input.value);
-
+    // Handle keyboard shortcuts
     switch (event.key) {
```

Comment:
re: when using keyboard navigation, please reselect the value.

Reply:
Can you explain more what is the issue? I do not get what you mean.

Reply:
Take a look here: https://ui.refinitiv.com/elements/time-picker, and try to ArrowUp/ArrowDown the field. The input value stays selected. If it is too difficult to implement for pagination, I am fine with as it is.

Reply:
Done